samples/samples-aspnet/embedded-auth-with-sdk/embedded-auth-with-sdk/Views/Manage/VerifyWebAuthnAuthenticator.cshtml

```diff
@@ -0,0 +1,70 @@
+﻿@model embedded_auth_with_sdk.Models.VerifyWebAuthnViewModel
+@{
+    ViewBag.Title = ViewBag.Title ?? "Verify your Web Authn authenticator.";
+}
+
+<hr />
+
+<div class="row d-center">
+    <div class="col-md-8">
+        <section id="forgotPasswordForm">
+            <h4>Verify your Web Authn authenticator.</h4>
+        </section>
+    </div>
+</div>
+
+<script type="text/javascript">
+    document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Since this is a sample application, we should be as explicit as possible. I guess the SIW has a lot of code we can just reuse but we're gonna include unnecessary logic that will overload and complicate the sample.

Reply:
Acknowledged.